packages/core/src/render3/instructions/control.ts

```diff
@@ -819,6 +848,8 @@ type ControlBindingKeys = Exclude<
  */
 type ControlBindings = {
   [K in ControlBindingKeys]?: unknown;
+} & {
+  classes: {[cls: string]: boolean};
```

Comment:
Can this not simply be:

```suggestion
  classes: {[cls: string]: boolean};
```